modules/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java

```diff
@@ -689,7 +690,8 @@ enum Operation {
         GET_BLOB_PROPERTIES("GetBlobProperties"),
         PUT_BLOB("PutBlob"),
         PUT_BLOCK("PutBlock"),
-        PUT_BLOCK_LIST("PutBlockList");
+        PUT_BLOCK_LIST("PutBlockList"),
+        BLOB_BATCH("BlobBatch");
```

Comment:
We can't be specific about the type of operation we're performing in a batch without inspecting the request body. I think it's better to track `BlobBatch` than potentially erroneously track `BatchDelete` (if one day we start using batch to "set access tier")